flask_monitoringdashboard/core/config/__init__.py.mongodb

```diff
@@ -0,0 +1,197 @@
+try:
```

Comment:
do we need this file? it's `__init__.py.mongodb`. Or did you add it for testing your local changes?

Reply:
I am using it to run the unit test actually. The unit test pipeline swaps the file to be sure to run against MongoDB. I would prefer to keep it for the time being

Reply:
I manage to remove that file with the latest refactoring